subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerJavaCachingSmokeTest.groovy

```diff
@@ -64,6 +71,8 @@ class AndroidSantaTrackerJavaCachingSmokeTest extends AbstractAndroidSantaTracke
         and:
         def expectedResults = agpVersion.startsWith('3.6')
             ? EXPECTED_RESULTS_3_6
+            : agpVersion.startsWith('4.0')
+            ? EXPECTED_RESULTS_4_0
             : EXPECTED_RESULTS
```

Comment:
```suggestion
                ? EXPECTED_RESULTS_4_0
                : EXPECTED_RESULTS
```

Reply:
IntelliJ formats this as in the PR, I'd rather not have a file that gets changed lines as soon as one formats it with IntelliJ

Reply:
I don't think optimizing for global reformatting should be a goal. At least for our Groovy and Java codebases we never did that, and I don't think we should start with this file here. As a result we have a lots and lots of files that'd be changed by IntelliJ if we reformatted them completely.

Reply:
Alrighty